src/ioc.ts

```diff
@@ -2,10 +2,12 @@ import { IocContainer } from '@tsoa/runtime'
 import { container } from 'tsyringe'
 
 import { logger, Logger, type ILogger } from './logger'
+import Database from './models/db/index.js'
 
 container.register<ILogger>(Logger, {
   useValue: logger,
 })
+container.register(Database, { useValue: new Database() })
```

Comment:
would need to pay attention to this if we add a new dep to the class

Reply:
yes